[2025-07-11T16:01:25.608+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-11T16:01:25.632+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_prediction_job.make_predictions scheduled__2025-07-11T15:55:00+00:00 [queued]>
[2025-07-11T16:01:25.641+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_prediction_job.make_predictions scheduled__2025-07-11T15:55:00+00:00 [queued]>
[2025-07-11T16:01:25.642+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-11T16:01:25.654+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): make_predictions> on 2025-07-11 15:55:00+00:00
[2025-07-11T16:01:25.664+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=716) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-11T16:01:25.666+0000] {standard_task_runner.py:72} INFO - Started process 718 to run task
[2025-07-11T16:01:25.667+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_prediction_job', 'make_predictions', 'scheduled__2025-07-11T15:55:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/prediction_job.py', '--cfg-path', '/tmp/tmpvirdr8ue']
[2025-07-11T16:01:25.670+0000] {standard_task_runner.py:105} INFO - Job 104: Subtask make_predictions
[2025-07-11T16:01:25.987+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_prediction_job.make_predictions scheduled__2025-07-11T15:55:00+00:00 [running]> on host 5a07e0ddeb4c
[2025-07-11T16:01:26.063+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_prediction_job' AIRFLOW_CTX_TASK_ID='make_predictions' AIRFLOW_CTX_EXECUTION_DATE='2025-07-11T15:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-11T15:55:00+00:00'
[2025-07-11T16:01:26.064+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-11T16:01:26.085+0000] {logging_mixin.py:190} INFO - Processing file: /opt/airflow/data/good_data/employee_data_with_errors_chunk_5.csv
[2025-07-11T16:01:26.099+0000] {logging_mixin.py:190} INFO - Unexpected error processing file /opt/airflow/data/good_data/employee_data_with_errors_chunk_5.csv: Out of range float values are not JSON compliant: nan
[2025-07-11T16:01:26.100+0000] {logging_mixin.py:190} INFO - Processing file: /opt/airflow/data/good_data/employee_data_with_errors_chunk_2.csv
[2025-07-11T16:01:26.112+0000] {logging_mixin.py:190} INFO - Unexpected error processing file /opt/airflow/data/good_data/employee_data_with_errors_chunk_2.csv: Out of range float values are not JSON compliant: nan
[2025-07-11T16:01:26.113+0000] {logging_mixin.py:190} INFO - Processing file: /opt/airflow/data/good_data/employee_data_with_errors_chunk_4.csv
[2025-07-11T16:01:26.120+0000] {logging_mixin.py:190} INFO - Unexpected error processing file /opt/airflow/data/good_data/employee_data_with_errors_chunk_4.csv: Out of range float values are not JSON compliant: nan
[2025-07-11T16:01:26.120+0000] {logging_mixin.py:190} INFO - Processing file: /opt/airflow/data/good_data/employee_data2_with_errors_chunk_20.csv
[2025-07-11T16:01:26.974+0000] {logging_mixin.py:190} INFO - Predictions for /opt/airflow/data/good_data/employee_data2_with_errors_chunk_20.csv completed successfully.
[2025-07-11T16:01:26.976+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-07-11T16:01:26.977+0000] {base.py:84} INFO - Retrieving connection 'processed_files_db'
[2025-07-11T16:01:26.983+0000] {logging_mixin.py:190} INFO - Saved processed files: ['employee_data_with_errors_chunk_5.csv', 'employee_data_with_errors_chunk_2.csv', 'employee_data_with_errors_chunk_4.csv', 'employee_data2_with_errors_chunk_20.csv']
[2025-07-11T16:01:26.984+0000] {logging_mixin.py:190} INFO - Total Predictions made: 1
[2025-07-11T16:01:26.985+0000] {python.py:240} INFO - Done. Returned value was: Predictions made for 4 files
[2025-07-11T16:01:27.007+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-11T16:01:27.008+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_prediction_job, task_id=make_predictions, run_id=scheduled__2025-07-11T15:55:00+00:00, execution_date=20250711T155500, start_date=20250711T160125, end_date=20250711T160127
[2025-07-11T16:01:27.085+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-11T16:01:27.283+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-11T16:01:27.285+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
